anaconda-pkg-build-lts/linux/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# Dockerfile for container images that Anaconda, Inc. uses to build conda
+# packages released on the "defaults" (repo.anaconda.com) channels.
+
+FROM continuumio/anaconda-pkg-build:2023.03.28
+
+RUN  /opt/conda/bin/conda install -c jcmorin-ana-org --quiet --yes anaconda-time-machine
```

Comment:
It would be slightly safer to do `conda install jcmorin-ana-org::anaconda-time-machine`. Doing this will force conda to install from this specific channel. If you use `-c ...`, conda can still use defaults.